res/layout/home_layout.xml

```diff
@@ -123,7 +124,8 @@
     <LinearLayout
         android:layout_width="fill_parent"
         android:layout_height="60dp"
-        android:gravity="center" >
+        android:gravity="center"
+        android:background="#000">
```

Comment:
for this background, I think we should create a color values or use @android:color/black